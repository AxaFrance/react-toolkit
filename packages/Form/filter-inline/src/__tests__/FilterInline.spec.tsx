import React from 'react';
import { render } from '@testing-library/react';

import FilterInline from '../FilterInline';

describe('<FilterInline />', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<FilterInline />);
    expect(asFragment()).toMatchSnapshot();
  });
});
