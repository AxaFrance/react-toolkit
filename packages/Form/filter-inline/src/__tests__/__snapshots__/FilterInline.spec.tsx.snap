// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterInline /> renders without crashing 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
